src/modules/events/stores/EventsStore.ts

```diff
@@ -468,6 +469,56 @@ export default class EventsStore implements IEventsStore {
 		});
 	};
 
+	@observable searchChannel: EventSource | null = null;
+
+	public changeToClosestEvent = (
+		searchDirection: SearchDirection.Previous | SearchDirection.Next,
+	) => {
+		if (this.searchChannel) this.searchChannel.close();
```

Comment:
- What if found event doesnt match current filter
- This looks weird. if timestampFrom = 12.00.00 and closest event is 11.59.59 we will just move 1 seconds backwards and reload whole tree from 11.59.59 ?

Reply:
If timestampFrom is 12.00.00 and closest event is 11.59.59 than we will move 1 second and we get new tree with 11.59.59 as timestampTo in range and (11.59.59 - interval) as timestampFrom